glitter/blockadmin/blocks.py

```diff
@@ -188,6 +188,33 @@ def continue_view(self, request, object_id):
             'change_url': change_url,
         })
 
+    def get_version(self, request):
+        # As this is the Django admin and we can't just set self.version_id due to thread safety
+        # issues, we need to find the version ID of the page when adding a new block. It'll be
+        # accessible in the URL - although this is a very hacky way to access it.
+        _, _, kwargs = resolve(request.path)
```

Comment:
Seemed like a good throwaway variable, however as Django convention uses `_` for translation - it's probably one to change.